### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by sreepadkarekkad**

### CODE REVIEW

File: api/src/main/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  #pattern:
+  #  file: %d | [${HOSTNAME}] | %clr(%5p) | [%t]   | %logger{36}:%L | %msg%n
+  #  console: %d | [${HOSTNAME}] | %clr(%5p) | [%t]   | %logger{36}:%L | %msg%n
+  level:
+    ca:
+      bc:
+        gov:
+          educ:
+            api:
+              graduation: ${APP_LOG_LEVEL}
```
Comment (infstar) [inline api/src/main/resources/application.yaml:34]: The package name for log level needs to be corrected.  ( graduation -> distribution )

File: api/src/test/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  #pattern:
+  #  file: %d | [${HOSTNAME}] | %clr(%5p) | [%t]   | %logger{36}:%L | %msg%n
+  #  console: %d | [${HOSTNAME}] | %clr(%5p) | [%t]   | %logger{36}:%L | %msg%n
+  level:
+    ca:
+      bc:
+        gov:
+          educ:
+            api:
+              graduation: INFO
```
Comment (infstar) [inline api/src/test/resources/application.yaml:38]: The package name needs to be corrected here too.

File: tools/jenkins/DeployToTest.groovy [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        label 'maven'
+    }
+    environment{
+        OCP_PROJECT = '77c02f-dev'
+        IMAGE_PROJECT = '77c02f-tools'
+        IMAGE_TAG = 'latest'
+        APP_SUBDOMAIN_SUFFIX = '77c02f-test'
+        APP_DOMAIN = 'apps.silver.devops.gov.bc.ca'
+        TAG = 'test'
+        REPO_NAME = 'educ-grad-graduation-api'
```
Comment (infstar) [inline tools/jenkins/DeployToTest.groovy:12]: Repository name is correct here?

File: api/src/main/resources/application.yaml (hunk 1 above)
Comment (sreepadkarekkad) [reply api/src/main/resources/application.yaml:34]: Ok..I have made this change :)

**Pull request merged**